static/js/sidebar.js

```diff
@@ -95,11 +95,19 @@ var SidebarView = Backbone.View.extend({
                     continue;
                 }
 
-                var entryHtml = this.sidebarItemTemplate({
-                    created: moment(markerModel.get("created")).format("LLLL"),
-                    type: SUBTYPE_STRING[markerModel.get("subtype")],
-                    icon: markerView.getIcon()
-                });
+                if (isRetina){
```

Comment:
I see what you mean. No problem.
I was contemplating whether I should create a new `entryHtml ` variable for the retina displays or just change the `icon` property, and for some reason I felt that choosing the later will be less readable in the future.
I'll do it ASAP.